src/app/layouts/ModalWindow.js

```diff
@@ -0,0 +1,34 @@
+export class ModalWindow {
+  constructor() {
+    const modalWindow = document.createElement('div');
+    modalWindow.classList.add('modalWindow');
+    modalWindow.id = 'modalWindow';
+
+    const parent = document.getElementById('swquiz-app');
+    parent.appendChild(modalWindow);
+  }
+
+  show(content) {
+    // ""Komponent umożliwia wyświetlenie okna modalnego z zawartością przekazaną jako inny komponent na wejściu do funkcji.""
+    // wiadomo jaki będzie typ komponentu jaki przyjdzie? Bo if (content) to chyba średni warunek...
```

Comment:
Nie rozumiem tutaj Twojego pytania?

Reply:
czy if(content) wystarczy? Czy zrobić coś typu if(typeof content == HTMLElement)?

Reply:
HTMLElement obejmuje wszystko co chcemy? Jesli tak to chyba dobra sprawa :) To jest minus JSa... W TS dalbys typ.